openslides/utils/autoupdate.py

```diff
@@ -133,50 +136,80 @@ def send_data(message):
                     {'text': json.dumps(output)})
 
 
-def inform_changed_data(instance, information=None):
+def inform_changed_data(instances, information=None):
     """
     Informs the autoupdate system and the caching system about the creation or
     update of an element.
+
+    The argument instances can be one instance or an interable over instances.
     """
-    try:
-        root_instance = instance.get_root_rest_element()
-    except AttributeError:
-        # Instance has no method get_root_rest_element. Just ignore it.
-        pass
-    else:
-        collection_element = CollectionElement.from_instance(
-            root_instance,
-            information=information)
-        # If currently there is an open database transaction, then the
-        # send_autoupdate function is only called, when the transaction is
-        # commited. If there is currently no transaction, then the function
-        # is called immediately.
-        transaction.on_commit(lambda: send_autoupdate(collection_element))
+    root_instances = set()
+    if not isinstance(instances, Iterable):
+        # Make surce instance is an iterable
+        instances = (instances, )
+    for instance in instances:
+        try:
+            root_instances.add(instance.get_root_rest_element())
+        except AttributeError:
+            # Instance has no method get_root_rest_element. Just ignore it.
+            pass
+
+    # Generates an collection element list for each root_instance.
```

Comment:
The comment does not match to the code. Do you want to create many collection element lists or only one collection element list that contains all root instances?